.babelrc

```diff
@@ -1,6 +1,6 @@
 {
   presets: [
-    "@babel/preset-env",
+    ["@babel/preset-env", { "modules": false }],
```

Comment:
I checked this was making something by running `yarn babel src/index.js` with the command line.